### PR TITLE
push notifications

### PR DESCRIPTION
**Pull request by faisalnaveedRNS**
## Description

Closes: #640 

Added push notifications api, read, unread api

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: big-dipper/imports/api/fcmtoken/server/methods.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            };
+          }
+    
+          return {
+            Code: StatusOk,
+            Message: Success,
+            Data: result,
+          };
+
+      // }).catch((e)=>{
```
Comment (mustafapylons) [inline big-dipper/imports/api/fcmtoken/server/methods.js:59]: are these for testing purpose, why is dead code still here?

File: big-dipper/imports/api/notifications/server/methods.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    const unSettled = Notifications.find({ settled: false });
+
+
+   
+    unSettled
+      .forEach((sale) => {
+        var sellerAddress = sale.from;
+        var saleID = sale._id;
+        var token;      
+        //get Firebase token for specieifed user address
```
Comment (mustafapylons) [inline big-dipper/imports/api/notifications/server/methods.js:135]: "specified" instead of "specieifed"

**Review by mustafapylons — COMMENTED** [review]
some issues, otherwise code is good.

**Review by faddat — APPROVED** [review]
thank you!

Love that you attended to the version numbers and such

**Comment by faddat** [comment]
blocks #653

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Pull request merged**